test/server/cards/13-CW/HallowedGround.spec.js

```diff
@@ -0,0 +1,126 @@
+            beforeEach(function() {
+                this.setupTest({
+                    phase: 'conflict',
+                    player1: {
+                        inPlay: ['shrine-maiden', 'shadowlands-hunter'],
+                        dynastyDiscard: ['hallowed-ground']
+                    },
+                    player2: {
+                        inPlay: ['righteous-magistrate'],
+                        honor: 11
+                    }
+                });
+                this.player1.placeCardInProvince('hallowed-ground', 'province 1');
+                this.maiden = this.player1.findCardByName('shrine-maiden');
+                this.hunter = this.player1.findCardByName('shadowlands-hunter');
+
+                this.magistrate = this.player2.findCardByName('righteous-magistrate');
+                this.startingHonor = this.player2.honor;
+                this.noMoreActions();
+            });
+
+            it('it should not do anything if the opponent does not have the air ring', function() {
+                this.initiateConflict({
+                    ring: 'fire',
+                    type: 'military',
+                    attackers: [this.maiden],
+                    defenders: []
+                });
+                this.noMoreActions();
+                expect(this.player1).toHavePrompt('Fire Ring');
+                expect(this.player2.honor).toBe(this.startingHonor - 1);
+            });
+
+            it('it should cause one more honor lost if they have the air ring', function() {
+                this.initiateConflict({
+                    ring: 'fire',
+                    type: 'military',
+                    attackers: [this.maiden],
+                    defenders: []
+                });
+                this.player2.claimRing('air');
+                this.noMoreActions();
+                expect(this.player1).toHavePrompt('Fire Ring');
+                expect(this.getChatLogs(3)).toContain('player2 loses 1 honor due to the constant effect of Hallowed Ground');
+                expect(this.player2.honor).toBe(this.startingHonor - 2);
+            });
+
+            it('should not working if righteous magistrate is defending', function() {
+                this.initiateConflict({
+                    ring: 'fire',
+                    type: 'military',
+                    attackers: [this.maiden, this.hunter],
+                    defenders: [this.magistrate]
+                });
+                this.player2.claimRing('air');
+                this.noMoreActions();
+                expect(this.player1).toHavePrompt('Fire Ring');
+                expect(this.player2.honor).toBe(this.startingHonor);
+            });
+
+            it('should cause honor loses for multiple conflict', function() {
+                this.initiateConflict({
+                    ring: 'fire',
+                    type: 'military',
+                    attackers: [this.hunter],
+                    defenders: []
```

Comment:
Can you add a defender here?  The purpose of a Shadowlands Hunter test would be to make sure "counts as unopposed" still triggers the condition even if the conflict itself is opposed.